flask_restful/inputs.py

```diff
@@ -242,12 +242,15 @@ def boolean(value):
     if type(value) == bool:
         return value
 
+    if value == '':
+        return None
+
     if not value:
         raise ValueError("boolean type must be non-null")
     value = value.lower()
-    if value in ('true', '1',):
+    if value in ('true', '1', 1):
```

Comment:
Good catch. I will add tests and update documentation if this functionality change is approved by maintainer. I don't want to spend a lot of time on this if it is not a feature that is wanted.